docs/install/docker-install/compose.md

```diff
@@ -90,18 +92,18 @@ docker compose -f flex-mysql-compose.yml ps
 To stop the containers.
 
 ```bash
-./down.sh
+./down.sh #You can pass mysql|postgres|ldap as an argument to the script. If you don't pass any, it will default to mysql.
 ```
 
 ## Configure Gluu flex
 
 1. Access the Docker container shell using:
 
     ```bash
-    docker compose -f flex-mysql-compose.yml exec flex sh #This opens a bash terminal in the running container
+    docker compose -f flex-mysql-compose.yml exec flex /bin/bash #This opens a bash terminal in the running container
```

Comment:
Shouldn't it using `exec -ti` to get interactive shell?

Reply:
Nope not necessarily. This will work.
![Screenshot (1546)](https://github.com/user-attachments/assets/7632be65-4376-4f7e-a0b0-da5da3b4b971)